modules/installation/pages/install-server-unified.adoc

```diff
@@ -73,3 +71,6 @@ For {susemgr} Server, the expected modules are:
 * Server Applications Module
 * Web and Scripting Module
 * SUSE Manager Server Module
+
+This completes installation of the SUMA Server. The next step is setting up of the server, as described here: xref:server-setup.adoc[]. 
```

Comment:
Couple of points here:

* We use one sentence per line (https://github.com/uyuni-project/uyuni-docs/wiki/markup-syntax#one-sentence-per-line)
* Try to never write SUMA or Uyuni, use {productname} everywhere, and if it only applies to one or the other, use an ifdef. (looks like I just discovered something new to document in our wiki!)
* We generally avoid result statements, as they don't tend to add value (https://github.com/uyuni-project/uyuni-docs/wiki/wug#results).
* Because we are moving towards more topic-based authoring, we try to avoid directional language (https://github.com/uyuni-project/uyuni-docs/wiki/wug#accessibility). Instead, use the "for ... see" construction (https://github.com/uyuni-project/uyuni-docs/wiki/wug#for-see).
* Always use the full reference in an xref, including the book name, just to prevent problems. It also presents better in PDF.

I would say something like this instead:

```suggestion
When you have finished installing the {productname} Server, you need to set it up so it is ready to use.
For more information, see xref:installation:server-setup.adoc[]. 
```